### PR TITLE
Fix for Android 13 in TypeText.kt so that tests with text input can work

### PR DESCRIPTION
**Pull request by Chog0**
This fix eliminates this issue https://github.com/avito-tech/avito-android/issues/1425

### CODE REVIEW

File: subprojects/android-test/ui-testing-core/src/main/kotlin/com/avito/android/test/element/field/actions/TypeText.kt [hunk 1]
```diff
@@ -75,7 +75,7 @@ internal class TypeText(private val stringToBeTyped: String) : ViewAction {
             ApplicationProvider.getApplicationContext<Application>()
                 .getSystemService(Context.INPUT_METHOD_SERVICE) as InputMethodManager
             )
-            .getFieldByReflectionWithAnyField("mIInputContext")
+            .getFieldByReflectionWithAnyField("mFallbackInputConnection")
```
Comment (dsvoronin) [inline subprojects/android-test/ui-testing-core/src/main/kotlin/com/avito/android/test/element/field/actions/TypeText.kt]: Hey, it should be under "SDK check if", right?

Comment (sergioboi) [reply subprojects/android-test/ui-testing-core/src/main/kotlin/com/avito/android/test/element/field/actions/TypeText.kt]: @Chog0 What do you think could you make changes with SDK check?

Comment (Chog0) [reply subprojects/android-test/ui-testing-core/src/main/kotlin/com/avito/android/test/element/field/actions/TypeText.kt]: Sure, will do that and ask you for review when it's ready.
Thanks for pointing this out, guys!

Comment (sergioboi) [reply subprojects/android-test/ui-testing-core/src/main/kotlin/com/avito/android/test/element/field/actions/TypeText.kt]: I will merge and create a release

Comment (dsvoronin) [reply subprojects/android-test/ui-testing-core/src/main/kotlin/com/avito/android/test/element/field/actions/TypeText.kt]: LGTM

Comment (sergioboi) [reply subprojects/android-test/ui-testing-core/src/main/kotlin/com/avito/android/test/element/field/actions/TypeText.kt]: Released in 2023.2

**Pull request closed without merging**